src/test/unit/web_interface/test_app_find_logs.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+import helperFunctions.fileSystem
+from test.unit.web_interface.base import WebInterfaceTest
+
+
+class IntercomMock:
+    @staticmethod
+    def get_backend_logs():
+        return ['String1', 'String2', 'String3']
+
+    def shutdown(self):
+        pass
+
+
+class TestShowStatistic(WebInterfaceTest):
+    def setUp(self, db_mock=None):
+        super().setUp(db_mock=IntercomMock)
+        self.config['Logging']['logFile'] = 'NonExistentFile'
```

Comment:
it would be better to move this line to the start of `test_logs_available()` to prevent possible side effects